guides/Diagnostics.md

```diff
@@ -0,0 +1,62 @@
+# Diagnostics
```

Comment:
You have created this page, but not added a link to it from the menu. I would also recommend going with lowercase for the filename.